TVShows/TVShows/Modules/ShowDetails/Cells/ShowDetailsImageCell.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ShowDetailsImageCell.swift
+//  TVShows
+//
+//  Created by Ivan Milicevic on 22/10/2018.
+//  Copyright © 2018 Ivan Milicevic. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+
+class ShowDetailsImageCell: UITableViewCell {
+
+    @IBOutlet weak var showDetailsImageView: UIImageView!
+
+    // MARK: - Private
+    private let placeholderImg: UIImage = UIImage(named: "missing")!
```

Comment:
Use underscore prefix for private outlets/variables

`private let _placeholderImg: UIImage = UIImage(named: "missing")!`

Also, never force unwrap `UIImage(named` method call since almost every iOS method which takes `UIImage` is using optional for `UIImage`